src/types.ts

```diff
@@ -36,6 +36,7 @@ export enum ContractEventsSignatures {
   ExtensionUninstalled = 'ExtensionUninstalled(bytes32,address)',
   ExtensionDeprecated = 'ExtensionDeprecated(bytes32,address,bool)',
   ExtensionUpgraded = 'ExtensionUpgraded(indexed bytes32,indexed address,uint256)',
+  ExtensionInitialised = 'ExtensionInitialised()',
```

Comment:
Not sure if this should belong here since it's an event specific to extension instance rather than the network (if my understanding is correct).

Reply:
Nah, don't worry about being so specific. This enum should do with all event names/sigs that we're dealing with in the block ingestor